hostinfo/hostinfo_linux.go

```diff
@@ -48,7 +48,17 @@ func linuxDeviceModel() string {
 	return ""
 }
 
+func getQnapQtsVersion(versionInfo string) string {
+	for _, field := range strings.Fields(versionInfo) {
+		if strings.HasPrefix(field, "QTSFW_") {
+			return "QTS " + field[6:]
```

Comment:
len("QTSFW_") also compiles to 6

or:

     if suffix := strings.TrimPrefix(field, "QTSFW_"); suffix != field {
             return "QTS "+suffix
     }

Reply:
Went with TrimPrefix, very nice.